site/docs/api.md

```diff
@@ -70,6 +72,27 @@ Iterable<CombinedScanTask> tasks = scan.planTasks();
 
 Use `asOfTime` or `useSnapshot` to configure the table snapshot for time travel queries.
 
+#### Row level
+
+Iceberg table scans start by creating a `ScanBuilder` object with `IcebergGenerics.read`.
+
+```java
+ScanBuilder scanBuilder = IcebergGenerics.read(table)
+```
+
+To configure a scan, call `where` and `select` on the `ScanBuilder` to get a new `ScanBuilder` with those changes.
+
+```java
+scanBuilder.where(Expressions.equal("id", 5))
+```
+
+When a scan is configured, call method `build` to execute scan. `build` return `CloseableIterable<Record>`
```

Comment:
Should this mention that it is Iceberg's `Record` class?

Reply:
Yep, I will add details.